apps/frontend/src/app/pages/person/gpt-chat/gpt-chat.component.ts

```diff
@@ -143,13 +168,17 @@ export class GptChatComponent implements OnInit {
           const messageError = codes.includes(error.statusCode)
             ? error.message
             : 'Error while generating chat completion';
+          const type = error.statusCode === 401 ? 'warning' : 'error';
           this.toastService.showToast({
             message: messageError,
-            type: 'error',
+            type: type,
```

Comment:
Type is not a very descriptive name, also, you could use it like `type`, instead of `type: type`